src/mmrelay/k8s_utils.py

```
@@ -0,0 +1,534 @@
+    """
+    required_keys = {
+        "namespace",
+        "storage_class",
+        "storage_size",
+        "connection_type",
+        "image_tag",
+    }
+    missing = sorted(required_keys - set(config))
+    if missing:
+        missing_keys = ", ".join(missing)
+        raise ValueError(f"Missing required config keys: {missing_keys}")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)